Code/Source/RobotImporter/URDF/JointsMaker.cpp

```diff
@@ -86,23 +102,48 @@ namespace ROS2
             { // Implemented as Hinge with limit set to -2*PI and 2*PI deg. Works fine in the Engine
                 jointComponent = followColliderEntity->CreateComponent<PhysX::EditorHingeJointComponent>();
                 followColliderEntity->Activate();
+
+                PhysX::EditorJointRequestBus::Event(
+                    AZ::EntityComponentIdPair(followColliderEntityId, jointComponent->GetId()),
+                    &PhysX::EditorJointRequests::SetVector3Value,
+                    PhysX::JointsComponentModeCommon::ParamaterNames::Rotation,
+                    rotation);
+
                 PhysX::EditorJointRequestBus::Event(
                     AZ::EntityComponentIdPair(followColliderEntityId, jointComponent->GetId()),
                     &PhysX::EditorJointRequests::SetLinearValuePair,
                     PhysX::JointsComponentModeCommon::ParamaterNames::TwistLimits,
                     PhysX::AngleLimitsFloatPair(AZ::RadToDeg(AZ::Constants::TwoPi), -AZ::RadToDeg(AZ::Constants::TwoPi)));
+
                 followColliderEntity->Deactivate();
             }
             break;
         case urdf::Joint::REVOLUTE:
             { // Hinge
                 jointComponent = followColliderEntity->CreateComponent<PhysX::EditorHingeJointComponent>();
                 followColliderEntity->Activate();
+
+                const double limit_upper = AZ::RadToDeg(joint->limits->upper);
```

Comment:
use naming standard here: limitUpper (you can also name it limitUpperDegrees for full clarity)